core/src/main/java/edu/wpi/grip/core/OperationDescription.java

```diff
@@ -105,29 +105,20 @@ public boolean equals(@Nullable Object o) {
 
     OperationDescription that = (OperationDescription) o;
 
-    if (name() != null ? !name().equals(that.name()) : that.name() != null) {
-      return false;
-    }
-    if (summary() != null ? !summary().equals(that.summary()) : that.summary() != null) {
-      return false;
-    }
-    if (category() != that.category()) {
-      return false;
-    }
-    if (icon() != null ? !icon().equals(that.icon()) : that.icon() != null) {
-      return false;
-    }
-    return aliases() != null ? aliases().equals(that.aliases()) : that.aliases() == null;
-
+    return name().equals(that.name())
+        && summary().equals(that.summary())
+        && category() == that.category()
+        && icon().equals(that.icon())
+        && aliases().equals(that.aliases());
   }
 
   @Override
   public int hashCode() {
-    int result = name() != null ? name().hashCode() : 0;
-    result = 31 * result + (summary() != null ? summary().hashCode() : 0);
-    result = 31 * result + (category() != null ? category().hashCode() : 0);
-    result = 31 * result + (icon() != null ? icon().hashCode() : 0);
-    result = 31 * result + (aliases() != null ? aliases().hashCode() : 0);
+    int result = name() == null ? 0 : name().hashCode();
```

Comment:
Don't use the getter methods, just access the fields directly.

`name == null ? 0 : name.hashCode()` can be replaced with `Objects.hashCode(name)`

This goes for lines 117 - 121